docs/MATEKSYS.md

```diff
@@ -28,3 +28,12 @@ Links to the MatekSys website are found below, which include product specificati
 The most convenient way of flashing the mLRS devices is via their USB-C port and using the [mLRS Web Flasher](https://mlrs.xyz/flash) app.
 
 Plug in the USB-C cable while pressing the bind button. This puts the mLRS device into DFU mode (LEDs are not flashing). Then follow the instructions provided by the mLRS Web Flasher app.
+
+## Tx Module | HC-04 Bluetooth Troubleshooting ##
+
+To use the HC-04 Bluetooth module on Matek mLRS Tx modules, no additional configuration should be needed as mLRS will automatically configure all of the necessary settings on the module. However, if you're having problems you can check the following items:
+
+- Ensure that the 3 dip switches are all in the 'ON' / left position when looking at the Tx module with the antenna pointing up. PShoto:
```

Comment:
what is this PShoto for?

Reply:
Well it was was supposed to be 'Photo' but just removed.